### PR TITLE
Add functionality to get any exchange rate

### PR DESCRIPTION
**Pull request by propyless**
Haven't tested the changes.. :P but I'm sure you will ;)
Default is SEK, but if you add
```
currency: GBP
```

then it should get you the exchange rate USD to GBP.

### CODE REVIEW

**Comment by propyless** [comment]
made sek to usd as default, and only 1 print

File: get_value.py [hunk 1]
```diff
@@ -7,14 +7,18 @@
 home = str(Path.home())
 config = load_config(f'{home}/.config/c20.yaml')
 num_tokens = config.get('num_tokens')
+currency = config.get('currency', 'USD')
+
```
Comment (marhag87) [inline get_value.py]: Omg extra new line, fix plx

File: get_value.py [hunk 2]
```diff
@@ -7,14 +7,18 @@
...
-sek = response.json().get('rates').get('SEK')
-
-
-print(int(value_per_token*num_tokens*sek))
+if currency.upper() == 'USD':
+  exchange_rate = 1
+else:
+  # Get dollar to chosen currency exchange rate
+  response = requests.get('https://api.fixer.io/latest?base=USD')
+  exchange_rate = response.json().get('rates').get(currency)
```
Comment (marhag87) [inline get_value.py]: Do a currency.upper() here as well to match the one on line 17

**Comment by propyless** [comment]
PTAL @marhag87

File: get_value.py (hunk 2 above)
Comment (propyless) [reply get_value.py]: added .upper to line 10 instead.

**Comment by marhag87** [comment]
Thanks for your contribution!

**Pull request merged**